ext/random/random.c

```diff
@@ -688,16 +688,6 @@ static PHP_GINIT_FUNCTION(random)
 }
 /* }}} */
 
-/* {{{ PHP_GSHUTDOWN_FUNCTION */
-static PHP_GSHUTDOWN_FUNCTION(random)
-{
-	if (random_globals->random_fd >= 0) {
-		close(random_globals->random_fd);
-		random_globals->random_fd = -1;
-	}
```

Comment:
`random_fd` should be removed from the globals struct, given it is unused now.

Reply:
Ah yes, I missed this when extracting this branch